### DISCUSSION TITLE
incremental collect mode gone?

### DISCUSSION THREAD

**Discussion by fokoenecke** [apache/kyuubi#1197] [Q&A] [answered]
I am using Kyuubi 1.3 and Spark 3.1.2. Unfortunately the data I want to retrieve is larger than the available memory. In Thrift there is a parameter to retrieve and forward data incrementally (`spark.sql.thriftServer.incrementalCollect`) and it seems to have been possible in Kyuubi too according to [this link](https://www.jianshu.com/p/e6893f4644ca) (`spark.kyuubi.operation.incremental.collect`).

Has the feature disappeared?

**Comment by ulysses-you** [comment]
Unfortunately, we don't support this feature now. cc @pan3793 @zhouyifan279 @timothy65535 if you are also interested in this.

**Comment by yaooqinn** [comment]
It looks good to me to bring it back

**Comment by pan3793** [comment, marked as answer]
I'm working on this feature, see #1198
